src/styles/blocks/card.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  &__style {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    font-family: inherit;
+  }
+  &__block {
+    display: flex;
+    flex-wrap: wrap;
+  }
+  &__form {
+    margin-top: 50px;
+    width: 198px;
+    height: 406px;
+    border: 1px solid #e5e5e5;
+    border-radius: 5px;
+  }
+  &__form:not(:first-child) {
+    margin-left: 48px;
+  }
+  &__block:nth-child(2) &__form {
+    margin-top: 46px;
+  }
+  &__image {
+    background-image: url(/images/imac.jpeg);
+    background-repeat: no-repeat;
+    background-size: 160px 134px;
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 40px;
+  }
+  &__title {
+    color: #060b35;
+    padding-left: 16px;
+    padding-right: 16px;
```

Comment:
you are settings this paddings in billion places, just set them to card and every child will be moved